aspnetcore/data/ef-rp/intro.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Razor Pages with Entity Framework Core in ASP.NET Core - Tutorial 1 of 8
+title: Entity Framework Core and Razor Pages - Tutorial 1 of 8
```

Comment:
to me it seems misleading to put EF first in the name:  It's really an RP tutorial that uses EF for data access.

Reply:
You might be optimizing for EF queries that don't mention RP at the expense of RP queries.  I would just do the EF --> Entity Framework change, but I think the search engines are smart enough to make that leap anyway.